src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaMetadataTrader.java

```diff
@@ -0,0 +1,57 @@
+package uk.ac.ebi.atlas.hcalandingpage;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.atlas.model.experiment.Experiment;
+import uk.ac.ebi.atlas.trader.ExperimentTrader;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+@Component
+public class HcaMetadataTrader {
+    private final ExperimentTrader experimentTrader;
+    private final HcaMetadataTraderDao hcaMetadataTraderDao;
+
+    public HcaMetadataTrader(ExperimentTrader experimentTrader,
+                             HcaMetadataTraderDao hcaMetadataTraderDao) {
+        this.experimentTrader = experimentTrader;
+        this.hcaMetadataTraderDao = hcaMetadataTraderDao;
+    }
+
+    public ImmutableMap<String, List<String>> getMetadata() {
+        var listOfMapsOfExperimentAccessionsAndOntologyIds =
```

Comment:
Is this a list of maps, a map of lists, or just a map?

Reply:
It is a set of list of maps.